extensions/guacamole-auth-openid/src/main/java/org/apache/guacamole/auth/openid/token/TokenValidationService.java

```diff
@@ -132,9 +124,89 @@ public String processUsername(String token) throws GuacamoleException {
             logger.debug("Malformed claim within received JWT.", e);
         }
 
+	return claims;
```

Comment:
Please correct the indentation here. The Guacamole source uses 4 spaces for each level and no tabs.